cmd/controller/main.go

```diff
@@ -31,5 +34,6 @@ func main() {
 	v1beta1.VirtualServiceUnmarshaler.AllowUnknownFields = true
 	v1beta1.GatewayUnmarshaler.AllowUnknownFields = true
 
-	sharedmain.Main("net-istio-controller", ingress.NewController, serverlessservice.NewController)
+	ctx := filteredFactory.WithSelectors(signals.NewContext(), network.SecretInformerLabelKey)
```

Comment:
I thought this would be remain optional - so it doesn't break existing users?

Reply:
Reason is that there is a comment from @evankanderson in the related ft doc:
"I'd hate to get to a point where (for legacy reasons) we have a huge amount of improvements in Knative that you need to "be in the in crowd" to know about and enable to get good behavior out of a cluster -- we should work to make this optimizations out of the box. "
If we make it optional we say that if you are in a big cluster you need to set X but in smaller ones your informers get all cluster stuff stored in mem. I dont think users like that either. 
In any case I dont mind making this optional but at some point this seems better to be on by default. Do have a process for a breaking change?